src/components/navBar.tsx

```diff
@@ -4,23 +4,25 @@ import Image from "next/image";
 import SiteButton from "./siteButton";
 import Link from "next/link";
 import { useState } from "react";
+import {
+  LinkedinShareButton,
+  FacebookShareButton,
+  TwitterShareButton,
+} from "next-share";
+import { FaCentercode } from "react-icons/fa6";
 
 export default function NavBar() {
-  const [signupClicked, setSignupClicked] = useState(false);
+  const [clickedButton, setClickedButton] = useState("");
 
-  function handleSignupClick() {
-    if (signupClicked === true) {
-      setSignupClicked(false);
-    } else {
-      setSignupClicked(true);
-    }
+  function handleNavButtonClick(e: any) {
+    setClickedButton(clickedButton === e.target.value ? "" : e.target.value);
   }
 
   return (
-    <div className="NavBar mx-auto flex h-fit w-[98%] justify-between border-b border-midnight/20 px-16 py-6">
```

Comment:
Also sad (even more so than the footer) that you got rid of the border here. I really feel like it "cheapens" the feel of the site for an unexplainable reason.